### DISCUSSION TITLE
Sendinblue

### DISCUSSION THREAD

**Discussion by mattiamarras** [chatwoot/chatwoot#4301] [Q&A] [answered]
Is possibile to user Sendinblue to receive and send emails?

Thanks.

M.

**Comment by vishnu-narayanan** [comment, marked as answer]
Look into the transaction emails feature offered in `Sendinblue`. Please configure the necessary environment emails with Chatwoot.
https://www.sendinblue.com/features/transactional-email/

**Reply by mattiamarras** [reply]
Thankyou, but I cannot develope the integration between Sendinblue and the action mailer in order to be able to receive emails on Chatwoot.

**Reply by sojan-official** [reply]
@mattiamarras  at the moment Sendinblue is not supported as an incoming email provider. We support the providers enabled by the action mailbox ref: https://guides.rubyonrails.org/action_mailbox_basics.html#configuration

unfortunately, the action mailbox doesn't support Sendinblue yet.
